src/main/java/com/e451/domain/HouseholdDemographic.java

```diff
@@ -0,0 +1,84 @@
+package com.e451.domain;
+
+public class HouseholdDemographic {
+
+    private Long hshdKey;
```

Comment:
Can this be called HouseholdKey? Being that it's the domain object, this is what the user will see.